### ISSUE TITLE
create template doesn't work

### ISSUE THREAD

**Issue by picli3** [yaqwsx/PcbDraw-Lib#17]
sorry to write at all times. make the change you indicated and I already have my PCB with the capacitors and resistors, however I try to create a new template to create a new model and I get this error
![image](https://user-images.githubusercontent.com/11140497/145913783-c2592d4d-7cfb-4c56-b7ab-670e8c3f506d.png)

I want to make all the components of this PCB, I hope you can help me.

![image](https://user-images.githubusercontent.com/11140497/145913858-d0b42edb-db69-4cb2-8bb9-e7f1492564e4.png)

**Comment by yaqwsx** [comment]
What does `create_template.py --help` say? It is weird that the option is not recognized - it is in the source code https://github.com/yaqwsx/PcbDraw-Lib/blob/b0a42a1cddd7441ea4b743c8ce922e911f9f2b36/scripts/create_template.py#L117

**Comment by picli3** [comment]
this is the message i get

![image](https://user-images.githubusercontent.com/11140497/146287104-dfb59318-1291-4a71-9635-03ff58fd9771.png)

**Comment by yaqwsx** [comment]
If you read the help, you will the problem - the script has two commands: one for building a template for every component on the board and one for building it out of a single footprint. And you did not specify this command. You simply miss the word `footprint`.

**Comment by picli3** [comment]
With this command it does not throw an error, but when I see the output it is only a red square.

`scripts/create_template.py footprint /usr/share/kicad/modules/Package_BGA.pretty/LFBGA-100_10x10mm_Layout10x10_P0.8mm.kicad_mod test/so2.svg`

![image](https://user-images.githubusercontent.com/11140497/146370984-1f81dc36-bad0-4ef2-9c62-dc71ec6da653.png)

**Comment by yaqwsx** [comment]
Could you share the footprint and resulting SVG?

**Comment by picli3** [comment]
this is the output file.

thanks for your help.

[so2.svg](https://drive.google.com/file/d/1SLy3_p9K0YaLX4YOgWtlTdPfO5N0mKM2/view?usp=sharing)

**Comment by yaqwsx** [comment]
![Screenshot from 2021-12-16 13-49-26](https://user-images.githubusercontent.com/1590880/146375161-d29be9c4-60b3-4f8a-957b-4edba2372d96.png)


It seems that you have created a template for the bottom view of the component. Please, try specifying the `--front` switch explicitly.

**Comment by picli3** [comment]
> sorry to write at all times. make the change you indicated and I already have my PCB with the capacitors and resistors, however I try to create a new template to create a new model and I get this error ![image](https://user-images.githubusercontent.com/11140497/145913783-c2592d4d-7cfb-4c56-b7ab-670e8c3f506d.png)


if specific that I get this error.

**Comment by yaqwsx** [comment]
You again miss the command `footprint`. You have to specify **both** `footprint` and `--front`. May I ask what lead you to the conclusion that you can omit the `footprint` part of the command?

**Comment by picli3** [comment]
you have it like this in the documentation, I don't see the `footprint` command 

`scripts/create_template.py --front \
  /usr/share/kicad/modules/Resistor_SMD.pretty/R_1206_3216Metric.kicad_mod \
  KiCAD-base/Resistor_SMD.pretty/R_1206_3216Metric.svg`

**Comment by yaqwsx** [comment]
Thanks for pointing that out. That is a mistake; the proper command is:
```
scripts/create_template.py footprint --front path/to/footprint.kicad_mod path/to/template.svg
```

**Comment by picli3** [comment]
I will try it when I get home. thank you!

**Comment by picli3** [comment]
now it worked, thank you very much

`scripts/create_template.py footprint --front /usr/share/kicad/modules/Package_BGA.pretty/LFBGA-100_10x10mm_Layout10x10_P0.8mm.kicad_mod test/so2.svg`

![image](https://user-images.githubusercontent.com/11140497/146470310-4593ce54-0ad2-46f1-9fa6-5b026a431979.png)

**Comment by yaqwsx** [comment]
For reference, the command was fixed in 47bd6d71dde1aa4563655a3bc03fac0eeb3de854
